### PR TITLE
document example 06

### PR DESCRIPTION
**Pull request by talSofer**
Docs suggestion

### CODE REVIEW

File: 06-airflow-dag-example/README.md [hunk 1]
```diff
@@ -1,15 +1,26 @@
-## lakeFS + Airflow Example
+## lakeFS + Github + Airflow - example
 
-### Run
+* Build a custom **CD process** to make your code and git metadata accessible for an Airflow DAG.
+* Use Airflow to orchestrate your ETL run.
+* Use [lakeFS Airflow provider](https://github.com/treeverse/airflow-provider-lakeFS) to:
+   * Version control your raw, intermediate and processed data
+   * Link code versions and the data generated by running them
+   * Link Airflow DAG code versions to the data generated during a DAG run
```
Comment (johnnyaug) [inline 06-airflow-dag-example/README.md:8]: Is this a prerequisites section, a description of what's to come, or simply instructions? The relation between these bullets and the next section is not clear.

I think it should be something like:
```suggestion
This sample will give you an idea of how you can use  the [lakeFS Airflow provider](https://github.com/treeverse/airflow-provider-lakeFS) to:
   * Version control your raw, intermediate and processed data.
   * Link between code versions and the data generated by running them.
```

The first two bullets are more like prerequisites or assumptions I guess?

File: 06-airflow-dag-example/README.md [hunk 2]
```diff
@@ -1,15 +1,26 @@
...
 
-1. Browse to Airflow in [http://localhost:8085/](http://localhost:8085/). User: _airflow_, password: _airflow_.
-
-1. Browse to the lakeFS UI in [http://localhost:8084/](http://localhost:8084/). User: AKIAIOSFODNN7EXAMPLE, password: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY.
+3. Browse to Airflow in [http://localhost:8085/](http://localhost:8085/). User: _airflow_, password: _airflow_.
 
-1. Run the _etl_ DAG in Airflow and observe the results in lakeFS.
+4. Browse to the lakeFS UI in [http://localhost:8084/](http://localhost:8084/). User: AKIAIOSFODNN7EXAMPLE, password: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY.
 
+5. Run the _etl_ DAG in Airflow and observe the results in lakeFS.
```
Comment (johnnyaug) [inline 06-airflow-dag-example/README.md:26]: What's changed here? Just the numbers?

**Review by johnnyaug — CHANGES_REQUESTED** [review]
Awesome chart! Suggested changes for the doc.

File: 06-airflow-dag-example/README.md (hunk 1 above)
Comment (talSofer) [reply 06-airflow-dag-example/README.md]: ok that's a nice way to think about it! changed it

**Review by johnnyaug — APPROVED** [review]

**Pull request merged**